2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -40,7 +40,41 @@ const myBooks = [
 ];
 
 function createBookList(books) {
-  // TODO your code goes in here, return the ul element
+  // create new elements and styling
+  const ul = document.createElement('ul');
+  ul.style.listStyle = 'none';
+  ul.style.display = 'flex';
+  ul.style.justifyContent = 'center';
+  ul.style.width = '1200px';
+  ul.style.margin = 'auto';
+
+  books.forEach((item, i) => {
+    const li = document.createElement('li');
+    const p = document.createElement('p');
+    p.textContent = `${item.title} - ${item.author}`;
+    li.style.width = '300px';
+    li.style.margin = '30px';
+    // condition of the background color .
+    if (item.alreadyRead === true) {
+      li.style.backgroundColor = 'green';
+    } else {
+      li.style.backgroundColor = 'red';
+    }
+    // creating images .
+    const img = document.createElement('img');
+    img.style.width = '200px';
+    if (i === 0) {
+      img.src = 'assets/the_design_of_everyday_things.jpg';
+    } else if (i === 1) {
+      img.src = 'assets/the_most_human_human.jpg';
+    } else img.src = 'assets/the_pragmatic_programmer.jpg';
```

Comment:
```suggestion
    if (i === 0) {
      img.src = 'assets/the_design_of_everyday_things.jpg';
    } else if (i === 1) {
      img.src = 'assets/the_most_human_human.jpg';
    } else {
      img.src = 'assets/the_pragmatic_programmer.jpg';
    }
```
It's better to keep the same if-else style for all conditions. This makes it easier to read